### PR TITLE
Add runtime error check

### PR DESCRIPTION
**Pull request by yanglu1225**
添加-fno-sanitize-recover=all，遇到undefined-behavior，程序退出，exitcode=1。

### CODE REVIEW

**Comment by yanglu1225** [comment]
```
=================== Test output for //tests:timer_test:
Running main() from gmock_main.cc
[==========] Running 2 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 2 tests from TimerTest
[ RUN      ] TimerTest.Basic
tests/timer_test.cc:77:50: runtime error: member call on null pointer of type 'cris::core::TimerReport'
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior tests/timer_test.cc:77:50 in 
================================================================================
```

**Comment by chenhao94** [comment]
@yanglu1225 https://github.com/cyfitech/cris-core/pull/27 may fix your ubsan test failure. Please let me know if that works.

**Comment by yanglu1225** [comment]
`https://github.com/cyfitech/cris-core/pull/27 may fix your ubsan test failure. Please let me know if that works.`

可以解决这个error。

File: bazel/rules.bzl [hunk 1]
```diff
@@ -5,6 +5,7 @@ CRIS_CXXOPTS = [
     "-Wextra",
     "-g3",
     "-std=c++20",
+    "-fno-sanitize-recover=all",
```
[the remark this answers is not in the document]
Comment (xkszltl) [reply bazel/rules.bzl:8]: 我觉得是

**Review by chenhao94 — APPROVED** [review]

**Pull request merged**